.github/workflows.src/release.tpl.yml

```diff
@@ -12,15 +12,13 @@ jobs:
     steps:
     - name: Determine package version
       shell: bash
-      env:
-        BRANCH: "${{ github.event.client_payload.branch }}"
-      run: echo ::set-output name=version::${BRANCH#releases/}
+      run: echo ::set-output name=version::${GITHUB_REF#releases/}
       id: whichver
 
     - name: Build
       uses: edgedb/edgedb-pkg/integration/linux/build/<< tgt.name >>@master
       env:
-        SRC_REF: "${{ github.event.client_payload.branch }}"
```

Comment:
I think `GITHUB_REF` has this format: `refs/heads/<branch-name>`

Reply:
Ah, alright: ${GITHUB_REF#refs/heads/}